benchmarks/cloudsuite3/inmem-analytics/run.sh

```diff
@@ -136,25 +132,27 @@ done
 rm -rf results
 mkdir -p results
 
-run_setup
+docker create --name movielens-data cloudsuite/movielens-dataset
 
 start_time=$(date +%s%N)
-docker run $PRIVILEGED $MAXMEM $MAXSWAP --ulimit nofile=90000:90000 $CPUSETS_CPU $CPUSETS_MEM --rm --volumes-from data $CONTAINER /data/ml-latest /data/myratings.csv $DMEM $EMEM > results/raw_results.txt
+docker run $PRIVILEGED $MAXMEM $MAXSWAP --ulimit nofile=90000:90000 $CPUSETS_CPU $CPUSETS_MEM --rm --volumes-from movielens-data $CONTAINER /data/ml-latest-small /data/myratings.csv $DMEM $EMEM > results/raw_results.txt
 end_time=$(date +%s%N)
 time_taken=$(echo "scale=9; ($end_time - $start_time)/1000000000" | bc)
 
-echo "in-memory-analytics,\"$NOTE\",$time_taken," `grep time results/raw_results.txt | sed 's/Benchmark execution time: //' | sed 's/ms//'` >> $RESULTS
+docker rm movielens-data
```

Comment:
Ideally, we would like to verify this command succeeds before proceeding and handle any errors appropriately.